.github/workflows/rebase.yml

```diff
@@ -50,6 +50,7 @@ jobs:
           token: ${{ secrets.GNU_PUSH_PAT }}
           repository: ${{ env.BINUTILS_REPO }}
           ref: ${{ env.ORIGIN_BRANCH }}
+          fetch-depth: 0
```

Comment:
It is quite interesting that it is possible to make rebase with fetch-depth 0

Reply:
`fetch-depth: 0` means "entire history".